### ISSUE TITLE
Set custom accessibilityLabel for Cosmos View

### ISSUE THREAD

**Issue by RahulMukherjee** [evgenyneu/Cosmos#196]
Hi Team,
I want to set/override the accessibilityLabel with my custom text. But i am unable to do that. Please let me know is there any way to set the accessibilityLabel with my custom text?

**Comment by ipv02** [comment]
Hi,
I've encountered the same issue – I need to set the accessibilityLabel and accessibilityTraits for the CosmosView, but I'm having trouble doing that.
